esx_service/utils/docker-volume-auth-schema.sql

```diff
@@ -0,0 +1,28 @@
+PRAGMA foreign_key = ON;
+
+CREATE TABLE tenants(
+  id TEXT PRIMARY KEY NOT NULL,
+  name TEXT,
+  description TEXT,
+  default_datastore TEXT
+);
+
+CREATE TABLE vms(
+  vm_id TEXT PRIMARY KEY NOT NULL,
+  vm_name TEXT,
+  tenant_id TEXT NOT NULL,
+  FOREIGN KEY(tenant_id) REFERENCES tenants(id)
+);
+
+CREATE TABLE privileges (
+  tenant_id TEXT NOT NULL,
+  datastore TEXT NOT NULL,
+  global_visibility INTEGER,
+  create_volume INTEGER,
+  delete_volume INTEGER,
+  mount_volume INTEGER,
+  max_volume_size INTEGER,
+  usage_quota INTEGER,
```

Comment:
if it's in bytes, please add a comment or name usage_quote_in_B (I think it's better to be in MB)